src/ViewRenderer.php

```diff
@@ -42,12 +39,11 @@ public function render(ViewInterface $view, string $template, array $parameters)
 
         $obInitialLevel = ob_get_level();
         ob_start();
-        /** @psalm-suppress InvalidArgument */
-        PHP_VERSION_ID >= 80000 ? ob_implicit_flush(false) : ob_implicit_flush(0);
+        ob_implicit_flush(false);
 
         try {
             /** @psalm-suppress PossiblyInvalidFunctionCall */
-            $renderer->bindTo($view)($template, $parameters);
+            $renderer->bindTo($view)();
```

Comment:
Why removing `$template, $parameters`?

Reply:
See above: passed to `$renderer` function via `use`.